src/main/java/io/anserini/index/IndexCollection.java

```diff
@@ -190,7 +177,7 @@ public IndexCollection(Args args) throws Exception {
     }
 
     final Directory dir = FSDirectory.open(Paths.get(args.index));
-    final IndexWriterConfig config = new IndexWriterConfig(getAnalyzer());
+    final IndexWriterConfig config = new IndexWriterConfig(getAnalyzer()).setCodec(new Lucene99Codec());
```

Comment:
`lucene99` is the default in Lucene 9.9, so I guess it is fine not to specify it manually.